### PR TITLE
OCPBUGS#18158: Removed Ingress and apiVIPs entries in sample vSphere …

### PR DESCRIPTION
**Pull request by dfitzmau**
[OCPBUGS-18158](https://issues.redhat.com/browse/OCPBUGS-18158)

Version(s):
4.14 and 4.13

Link to docs preview:
* [Sample install-config.yaml file for VMware vSphere- UPI docs only](https://64295--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-restricted-networks-vsphere#installation-vsphere-config-yaml_installing-restricted-networks-vsphere)
* [Additional VMware vSphere configuration parameters](https://64295--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installation-config-parameters-vsphere#installation-configuration-parameters-additional-vsphere_installation-config-parameters-vsphere)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-vsphere-config-yaml.adoc [hunk 1]
```diff
@@ -42,8 +42,6 @@ networking:
 ---
 platform:
   vsphere:
-    apiVIPs:
```
Comment (dfitzmau) [inline modules/installation-vsphere-config-yaml.adoc:45]: As I'm removing these fields from the UPI docs, I should update the [Additional VMware vSphere configuration parameters](https://64295--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installation-config-parameters-vsphere#installation-configuration-parameters-additional-vsphere_installation-config-parameters-vsphere) table as it lists the `apiVIPs` and `ingressVIPs` YAML entries. Update them in regard to stating that they apply to the IPI `install-config.yaml` file ?

Comment (dfitzmau) [reply modules/installation-vsphere-config-yaml.adoc:45]: @WenXinWei and @rvanderp3

Comment (WenXinWei) [reply modules/installation-vsphere-config-yaml.adoc:45]: i agree. You can add a description to indicate that they are used for ipi.

Comment (dfitzmau) [reply modules/installation-vsphere-config-yaml.adoc:45]: Thanks, @WenXinWei . Updated.

File: modules/installation-vsphere-config-yaml.adoc [hunk 2]
```diff
@@ -42,8 +42,6 @@ networking:
 ---
```
Comment (nalhadef) [inline modules/installation-vsphere-config-yaml.adoc:42]: Up above: There should be a content type before the primary title in Line 15.

**Comment by nalhadef** [comment]
Hi, Darragh.

Just one small comment. Otherwise, /lgtm.

File: modules/installation-vsphere-config-yaml.adoc (hunk 2 above)
Comment (dfitzmau) [reply modules/installation-vsphere-config-yaml.adoc:43]: Sorry. I'm only seeing this now. I'll make this update through another open PR. This is clearly a legacy docs update.

**Pull request merged**